lib/KCard/index.vue

```diff
@@ -0,0 +1,314 @@
+<template>
+
+  <BaseCard
+    :to="to"
+    :title="title"
+    :headingLevel="headingLevel"
+    :titleLines="titleLines"
+    :style="wrapperStyle"
+  >
+    <template #title>
+      <!-- @slot Optional slot section containing the title contents, should not contain a heading element. -->
+      <div class="title-slot">
+      <!-- <slot name="title"></slot> -->
+      </div>
+    </template>
+
+    <template #default>
+      <aside
+        v-if="thumbnailDisplay !== 'none'"
+        :style="{ dynamicOrder }"
+        :class="{
+          'image-class-vertical': layout === 'vertical',
+          'image-class-horizontal': (layout === 'horizontal' && thumbnailDisplay === 'small')
+        }"
+      >
+        <KImg
+          v-if="thumbnailSrc"
+          :src="thumbnailSrc"
+          :isDecorative="true"
+          :appearanceOverrides="{ scaleType: thumbnailScaleType, width: cardContentPartial }"
+          class="thumbnail-image"
+          :style="imageRadius"
+        />
+        <slot v-if="!thumbnailSrc" name="thumbnailPlaceholder"></slot>
+      </aside>
+      <div class="display-inline-block" :style="cardContentPartial">
+        <div
+          data-testid="aboveTitle"
+          class="above-title-style"
+          :class="{ 'above-title-horizontal': layout === 'horizontal' }"
```

Comment:
I dont see any CSS rules defining none of the classes: above-title-horizontal, title-horizontal, below-title-horizontal, so we will need to remove them from the nodes also.